AnyText/README.md

```diff
@@ -0,0 +1,92 @@
+# AnyText Framework
+
+
+
+## Usage
+
+
+
+## The Language Server Protocol (LSP)
+https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/
+
+
+### Features
+
+* **Folding:** Enables certain parts of the document like comments, classes or other custom 
+ranges to be folded away (hidden).
+* **Selection:** Enables the selection (VSCode default: Alt + Shift + Right Arrow) of symbols 
+and the surrounding scope.
+* **Goto Definition:** Enables jumping to the definition of a symbol in the document.
+* **Find References:** Enables the viewing of all references of a symbol in a document.
+* **Highlights:** Enables highlighting of matching symbols (see find references) and textual 
+occurrences in a document.
+* **Rename:** Enables the renaming of a symbol in a document (see find references).
+* **Document Symbols/Outline:** Enables the symbols of the document to be shown in the outline 
+(VSCode default left-hand side and top of document). The outline at the top shows the current 
+position in the document and both outlines can be used to jump between symbols in the document.
+
+### Feature configuration
+
+While the features were implemented generically, some of them can be configured or require 
+manual configuration to function properly. Manual configurations can be added to the rules 
+in the **\<GrammarName\>.manual.cs** file located under */AnyText/Grammars*.
+
+**Folding:** 
+Folding can be configured in multiple ways. A folding range can be a comment, a number of 
+imports/includes statements, a region or a generic range. While comments don't need to be 
+configured manually, the default implementations of the remaining kinds of folding ranges 
+can and often should be overridden. Overrides need to be done for each rule individually.
+
+* **public override bool IsRegionStartLiteral(string literal)**: 
```

Comment:
Code wird in GitHub-flavoured Markdown in Backticks gesetzt, aber fett geht auch